websites/A/Animekimi/presence.ts

```diff
@@ -138,6 +141,22 @@ presence.on("UpdateData", async () => {
     delete presenceData.startTimestamp;
     delete presenceData.endTimestamp;
   }
+  if (!time) {
+    delete presenceData.startTimestamp;
+    delete presenceData.endTimestamp;
+  }
+  if (privacy) {
+    delete presenceData.details;
+    delete presenceData.state;
+  }
```

Comment:
Basically show nothing? I wouldn't want it to show nothing even If I want privacy